libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -14,8 +14,8 @@ userdata Location method get_distance float Location
 userdata Location method offset void float'skip_check float'skip_check
 userdata Location method offset_bearing void float'skip_check float'skip_check
 userdata Location method offset_bearing_and_pitch void float'skip_check float'skip_check float'skip_check
-userdata Location method get_vector_from_origin_NEU boolean Vector3f'Null
-userdata Location method get_vector_from_origin_NEU depends AP_AHRS_ENABLED
+userdata Location method get_vector_from_origin_NEU_cm boolean Vector3f'Null
+userdata Location method get_vector_from_origin_NEU_cm depends AP_AHRS_ENABLED
```

Comment:
You could rename this back to the original method for use in scripts. Currently this is a breaking change.

Reply:
> You could rename this back to the original method for use in scripts. Currently this is a breaking change.

Can we create a clone (or shim?) with the old method name that scripts can use but get a deprecation warning?

At some stage in the (far-flung) future it would be nice to re-introduce the method but working in metres rather than centimetres.

Reply:
> Can we create a clone (or shim?) with the old method name that scripts can use but get a deprecation warning?

Yeah, but the binding generator doesn't currently support two names for the same function, you could shim or do a manual binding.

Reply:
> > Can we create a clone (or shim?) with the old method name that scripts can use but get a deprecation warning?
> 
> Yeah, but the binding generator doesn't currently support two names for the same function, you could shim or do a manual binding.

Thanks, will look at that.